### ISSUE TITLE
PEP 0484 compatibility

### ISSUE THREAD

**Issue by lig** [magniff/watch#1]
It would be nice to have this compatible with Type Hints defined in [PEP 0484](https://www.python.org/dev/peps/pep-0484/)

**Comment by magniff** [comment]
@lig As far as I know this pep accepted for 3.5+, that is considered to be too 'bleeding edge' for now.

**Comment by lig** [comment]
Right, it's about `Python-Version: 3.5` and it will be good to have something compatible with the current recommended implementation for attrs validation.
As usual there is a backport available on PyPI: [typing](https://pypi.python.org/pypi/typing)

**Comment by magniff** [comment]
@lig added at least [some support](https://github.com/magniff/watch#how-to-control-function-arguments-and-result).

**Comment by lig** [comment]
Any news on this one?

**Comment by magniff** [comment]
Hmm, personally I am not really into it, so if you feel like that kind of functionality is essential for you, you are welcome to either fork this repo or submit a MR.
